Deferred/Deferred.swift

```diff
@@ -110,15 +178,48 @@ public struct Deferred<Value> {
     main queue immediately. The function is always executed asynchronously, even
     if the Deferred is filled from the main queue.
     
-    :param: function A function that uses the determined value.
+    :param: body A function that uses the determined value.
     */
-    public func uponMainQueue(function: Value -> ()) {
-        upon(dispatch_get_main_queue(), function: function)
+    public func uponMainQueue(body: Value -> ()) {
+        upon(dispatch_get_main_queue(), body: body)
+    }
+
+    /**
+    Waits synchronously for the value to become determined.
+
+    If the value is already determined, the call returns immediately with the
+    value.
+
+    :param: time A length of time to wait for the value to be determined.
+    :returns: The determined value, if filled within the timeout, or `nil`.
+    */
```

Comment:
#33 completes the move to the new comment style. The `///` style is preferred by Swift 2. It triggers the new keyword syntax (` - returns:`) and has much clearer indentation style.